edge/msghub/cpu2msghub/Makefile

```diff
@@ -58,6 +58,12 @@ publish-all-arches:
 	ARCH=arm64 $(MAKE) build-test-publish
 	hzn exchange pattern publish -f horizon/pattern-all-arches.json
 
+icp-publish:
```

Comment:
Sorry i didn't mention this before, but i think we will end up needing this target on the ibm public cloud too. Can you change the name of the target to something like publish-only ?

Reply:
Will do!